lib/scheduler.pm

```diff
@@ -125,7 +150,8 @@ sub load_yaml_schedule {
         while (my ($var, $value) = each %schedule_vars) { set_var($var, $value) }
         my @schedule_modules = parse_schedule($schedule);
         parse_test_suite_data($schedule);
-        loadtest($_) for (@schedule_modules);
+        my $args = OpenQA::Test::RunArgs->new();
```

Comment:
I don't think that such approach would improve anything, as you create new object and we need to get already defined context or define it, but not just create empty object. Take a look on `wicked::TestContext`. So in case you need something like that, we should define which context we need and then pass it to the loadtest.

Reply:
i focused on the modules loaded on the publiccloud. I just went one step ahead to include the wicked modules i found using the `RunArgs`. in this regards this would work. But you are right. a complete job here would be to take in account other things, like `wicked::TestContext`. i am thinking to add another key variable in the yaml, like `test_context: OpenQA::Test::RunArgs`. i assume also that one one of the *context* objects is used per test. what do you think about this approach?

Reply:
i am closing this and check latest commit feefbf0